net/third_party/quiche/src/quiche/quic/core/crypto/crypto_protocol.h

```diff
@@ -194,8 +194,26 @@ const QuicTag kMAD0 = TAG('M', 'A', 'D', '0');   // Ignore ack delay
 const QuicTag kMAD2 = TAG('M', 'A', 'D', '2');   // No min TLP
 const QuicTag kMAD3 = TAG('M', 'A', 'D', '3');   // No min RTO
 const QuicTag k1ACK = TAG('1', 'A', 'C', 'K');   // 1 fast ack for reordering
+#if defined(USE_COBALT_CUSTOMIZATIONS)
+const QuicTag kAKD1 = TAG('A', 'K', 'D', '1');   // 10 packets received before
+                                                 // acking
+const QuicTag kAKD2 = TAG('A', 'K', 'D', '2');   // 20 packets received before
+                                                 // acking
+#endif  // defined(USE_COBALT_CUSTOMIZATIONS)
 const QuicTag kAKD3 = TAG('A', 'K', 'D', '3');   // Ack decimation style acking
```

Comment:
Why do we create tags similar to kAKD3 when we're basing our rate on number of packets instead of RTT? Shouldn't we create a new string of letters?

Reply:
Ah, I guess it needs to be exactly 4 chars, so that's out too. This is fine then